### DISCUSSION TITLE
MLM2Pro Connection Issues

### DISCUSSION THREAD

**Discussion by bmurf07** [springbok/MLM2PRO-GSPro-Connector#116] [Q&A]
I'm having trouble with the connector on an HP OMEN PC.
The connector has worked twice (flawlessly when it connects, BTW) out of a few dozen tries over several days.

Today it connected for about 1 minute, then the connector said the "remote device closed the connection."  
Since then is will not connect, usually saying to check if the device is on and in range.

I have restarted all devices multiple times, reinstalled the connector, etc.
The MLM app on my iphone connects every time and Awesome Golf Authentication is always successful.

Attached are the logs from todays attempt...thanks for any help..very frustrating!

[mlm2pro-gspro-connect.log](https://github.com/user-attachments/files/17081520/mlm2pro-gspro-connect.log)

**Comment by wonder99** [comment]
My best guess is the Bluetooth strength. Try putting the MLM directly beside the PC and see if it improves.
Most people use an external Bluetooth antenna because there's so much noise near the PC and the mlm's antenna is weak.
I use a powered usb extender, and a usb/Bluetooth adapter at the end.
When you do get a connection, let us know the RSSI signal strength

**Reply by bmurf07** [reply]
For testing I have the MLM sitting on the desk right next to the PC...no luck.
RSSI: -54 (green)

**Comment by wonder99** [comment]
Oh you should also try launching gspro before the connector, so that it doesn't have to. I find I need to point the connector to GSPro.exe, and not it's normal launcher

**Reply by bmurf07** [reply]
Yes, I'm not auto-starting anything at this point. It works whenever the MLM connects, but not getting that far right now.

**Comment by wonder99** [comment]
Hmm.
In think you might want to try another Bluetooth adapter. This is the one I
use. Hopefully you can buy from a place that accepts returns of it doesn't
help.

UGREEN Bluetooth Adapter for PC, Long Range Bluetooth 5.3 USB Dongle
100m/328ft, 5-Devices Paired Bluetooth Transmitter Receiver for Wireless
Keyboard,Mouse,Speaker,Headset, Support Windows 11/10/8.1
https://a.co/d/0J1wJ2d

Btw, because of the long distance between the MLM and my PC, I use this
extender as well
Alxum 32 Feet Active USB...
https://www.amazon.ca/dp/B094QNV1MJ?ref=ppx_pop_mob_ap_share
